Makefile

```diff
@@ -565,7 +579,11 @@ ${WRKDIR}/.efiboot_done:
 .if !defined(NO_EFIBOOT)
 	@echo -n "Creating EFI boot image ..."
 	${_v}${MKDIR} -p ${WRKDIR}/efiroot/EFI/BOOT
-	${_v}${CP} ${WRKDIR}/cdboot/${EFILOADER} ${WRKDIR}/efiroot/EFI/BOOT/BOOTX64.efi
+.	if ${TARGET} == aarch64
```

Comment:
Here we also should use a variable for the bootfile instead of the complex if-else forking.

Reply:
Make sesne, added EFIBOOT for that.